code/modules/projectiles/guns/projectile/automatic/straylight.dm

```diff
@@ -1,6 +1,10 @@
 /obj/item/weapon/gun/projectile/automatic/straylight
 	name = "FS SMG .35 Auto \"Straylight\""
-	desc = "A compact and lightweight submachinegun that sprays small rounds rapidly. Sacrifices a fire selector to cut mass, so it requires a careful hand. Uses .35 Auto rounds."
+	desc = "A compact and lightweight submachinegun that sprays small rounds rapidly. It supposed to be testing ground for newly \
```

Comment:
"A compact, lightweight rapid submachine gun. Primarily used for testing ammunition and weapon modifications. Suffers from poor recoil control, and underperforming ballistic impact, but makes up for this through sheer firerate. Uses .35 Auto rounds."

Reply:
Encorporated.